collectors/APICallCollector.js

```diff
@@ -29,20 +29,19 @@ class APICallCollector extends BaseCollector {
      */
     async addTarget({cdpClient, url}) {
         const trackerTracker = new TrackerTracker(cdpClient.send.bind(cdpClient));
+        try {
```

Comment:
I think this was on purpose at the end. W/o bindings set up and Debugger.paused listener brakpoints will fire but not be caught (potentially becoming paused forever?). I think you mentioned that this didn't change antything significantly, so I'd rather revert it.

Reply:
I think you're referring to a different thing here. This change was introduced in https://github.com/duckduckgo/tracker-radar-collector/pull/90/commits/826058696bcf5c9883df0d8620f8516ec98c9b22, I will roll back